SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/service/SubscriptionService.java

```diff
@@ -578,8 +600,11 @@ && shouldSendOut(potentialSub, message.getPayload())) {
 			case AppConstants.DELETE_ATTRIBUTE_REQUEST -> {
 
 				if (shouldFire(Sets.newHashSet(message.getAttribName()), potentialSub)) {
-					unis.add(localEntityService.getAllByIds(message.getTenant(), message.getId(), true).onItem()
+					unis.add(localEntityService.getAllByIds(message.getTenant(), message.getId(), true,potentialSub.getSubscription().isLocalOnly()).onItem()
```

Comment:
Add postman test cases for subscription local only functionality.

Reply:
done